packages/api/tests/endpoints/test-granules.js

```diff
@@ -107,17 +131,104 @@ async function setupBucketsConfig() {
   return { internalBucket: systemBucket, publicBucket: buckets.public.name };
 }
 
-// create all the variables needed across this test
-let esClient;
-let esIndex;
-let accessTokenModel;
-let granuleModel;
-let collectionModel;
-let jwtAuthToken;
+/**
+ * Helper for creating a granule, and files belonging to that granule
+ * @param {Knex} dbClient - Knex client
+ * @param {number} collectionCumulusId - cumulus_id for the granule's parent collection
+ * @param {boolean} published - if the granule should be marked published to CMR
+ * @returns {Object} fake granule object
+ */
+async function createGranuleAndFiles(dbClient, collectionCumulusId, published) {
+  s3Buckets = {
```

Comment:
General code style nit: rather than having this be set by overriding a global `s3Buckets` variable, I would just assign it directly in this helper. And then return the buckets with the newGranule info in the function return. This would also avoid the need for tests to be serial to run or cleanup correctly

```suggestion
  const s3Buckets = {
```

Reply:
👍 yup, sure thing. https://github.com/nasa/cumulus/pull/2057/commits/bbf2fb28087ce20890fb6e30709a932b1dd42a8d